src/presentation/http/router/note.test.ts

```diff
@@ -27,38 +26,43 @@ describe('Note API', () => {
         url: '/note/resolve-hostname/codex.so',
       });
 
-      expect(response?.statusCode).toBe(expectedStatus);
+      expect(response?.statusCode).toBe(200);
 
       expect(response?.json()).toStrictEqual(expectedResponse);
     });
 
     test('Returns 404 when note not found', async () => {
-      const expectedStatus = 404;
-
       const response = await global.api?.fakeRequest({
         method: 'GET',
         url: '/note/resolve-hostname/incorrect_hostname',
       });
 
-      expect(response?.statusCode).toBe(expectedStatus);
+      expect(response?.statusCode).toBe(404);
 
       expect(response?.json()).toStrictEqual({ message: 'Note not found' });
     });
   });
 
   describe('GET note/:notePublicId ', () => {
     test('Returns note by public id with 200 status when note is publicly available', async () => {
-      const expectedStatus = 200;
       const correctID = 'Pq1T9vc23Q';
       const expectedResponse = {
         'note': {
-          'id': 2,
+          'id': 3,
           'publicId': 'Pq1T9vc23Q',
           'creatorId': 1,
           'content': null,
           'createdAt': '2023-10-16T13:49:19.000Z',
           'updatedAt': '2023-10-16T13:49:19.000Z',
         },
+        'parentNote': {
+          'id': 4,
+          'publicId': '73NdxFZ4k7',
+          'creatorId': 1,
+          'content': null,
+          'createdAt': '2023-10-16T13:49:19.000Z',
+          'updatedAt': '2023-10-16T13:49:19.000Z',
+        },
```

Comment:
unexpected lines. This test checks for returning a specific not by its id. Test title says nothing about parentNote requirement. Its better to create separate test for checking parent-note